libs/langchain/langchain/prompts/base.py

```diff
@@ -99,6 +100,20 @@ def check_valid_template(
         )
 
 
+def get_template_variables(template: str, template_format: str) -> List[str]:
```

Comment:
Is there a test that verifies that repeated variables only appear once for each template format?